src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingSpec.cs

```diff
@@ -429,59 +545,55 @@ public void ClusterSharding_should_use_second_node()
                     r.Tell(new Counter.EntityEnvelope(2, Counter.Increment.Instance));
                     r.Tell(new Counter.Get(2));
                     ExpectMsg(3);
-                    Assert.Equal(Node(_second) / "user" / "counterRegion" / "2" / "2", LastSender.Path);
+                    LastSender.Path.Should().Be(Node(_config.Second) / "user" / "counterRegion" / "2" / "2");
 
                     r.Tell(new Counter.Get(11));
                     ExpectMsg(1);
-                    // local on first
-                    Assert.Equal(r.Path / "11" / "11", LastSender.Path);
+                    var path11 = LastSender.Path;
+                    LastSender.Path.ToStringWithoutAddress().Should().Be((r.Path / "11" / "11").ToStringWithoutAddress());
```

Comment:
had to change the spec here.
When the shard "11" is being created, the sharding status is: 
- first node: ["1"]
- second node: ["2"]
Since LeastShardAllocationStrategy has no ordering between nodes with the same number of shards, the next shard could be placed to either of them, right? 
Not sure why akka assumes that shard "11" should be local.